integration-test/src/test/java/org/ehcache/integration/statistics/TierCalculationTest.java

```diff
@@ -267,4 +276,60 @@ private void innerClear() {
     tierStatistics.clear();
     changesOf(-1, -4, -3, -2, -1);
   }
+
+  @Test
+  public void testMappingCount() {
+    assertThat(tierStatistics.getMappings()).isEqualTo(0);
+    cache.put(1, "a");
+    assertThat(tierStatistics.getMappings()).isEqualTo(1);
+  }
+
+  @Test
+  public void testMaxMappingCount() {
+    assertThat(tierStatistics.getMaxMappings()).isEqualTo(-1); // FIXME Shouldn't it be 0?
+    cache.put(1, "a");
+    cache.put(2, "b");
+    cache.remove(1);
+    assertThat(tierStatistics.getMappings()).isEqualTo(1); // FIXME: I was expecting 2
+  }
+
+  @Test
+  public void testAllocatedByteSize() {
+    assumeFalse(tierName.equals("OnHeap")); // FIXME: Not calculated for OnHeap when a size is allocated
+
+    long size = tierStatistics.getAllocatedByteSize();
+    cache.put(1, "a");
+    assertThat(tierStatistics.getAllocatedByteSize()).isGreaterThan(size); // FIXME: Why is allocated growing?
+  }
+
+  @Test
+  public void testOccupiedByteSize() {
+    assertThat(tierStatistics.getOccupiedByteSize()).isEqualTo(0);
+    cache.put(1, "a");
+    assertThat(tierStatistics.getOccupiedByteSize()).isGreaterThan(0);
+  }
+
+  @Test
```

Comment:
ok i guess I will check the same then for the TMS system tests.  I was just curious as I had some issues testing eviction counts in tests, since they are "random".  Its just a mystery I guess.